lib/html_tag.js

```diff
@@ -30,6 +30,11 @@ function htmlTag(tag, attrs, text, escape = true) {
   }
 
   if (escape && text && tag !== 'style') text = escapeHTML(String(text));
+  if (text && tag === 'style') {
+    text = text.replace(/url\(['"](.*?)['"]\)/gi, (urlAttr, url) => {
+      return `url("${encodeURL(url)}")`;
```

Comment:
this is necessary (instead of `return encodeURL(urlAttr)`) to avoid encoding single/double quote.